index.html

```diff
@@ -0,0 +1,764 @@
+          <p>The incoming president thousands of asylum-seekers living in limbo in crowded </p>
+        </div>
+        <hr>
+      </div>
+
+      <div class="more-stories-card ">
+        <a href="#"><img class="m-2 float-start" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711659/qanon-slogan-rally.jpg?w=95&amp;h=95&amp;f=bbc37f8fca978991a41d9934008c01b7" alt="Immigrant Challenge"></a>
+        <div>
+          <h3 class="m-2"><a href="#" class="text-dark text-decoration-none"> Biden's Biggest Challenge? 67,000 Migrants Stranded at U.S.-Mexico Border</a></h3>
+        <p>The incoming president thousands of asylum-seekers living in limbo in crowded shelters and makeshift </p>
+      </div>
+      <hr>
+    </div>
+
+    <div class="more-stories-card">
+      <a href="#"><img class="m-2 float-start" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711234/migrant-child-us-mexico-border-stranded-trump.jpg?w=95&amp;h=95&amp;f=4cde0203f23604897832a6d5ad1d559f" alt="Immigrant Challenge"></a>
+      <div>
+        <h3 class="m-2"><a href="#" class="text-dark text-decoration-none"> Biden's Biggest Challenge? 67,000 Migrants Stranded at U.S.-Mexico Border</a></h3>
+        <p>The incoming president thousands of asylum-seekers living in limbo in crowded  </p>
+    </div>
+    <hr>
+  </div>
+      </div>
+    </div>
+    </div>
+
+<!-- END OF COLUMN TWO -->
+
+<!-- COLUMN 3 -->
+
+      <div class="column3 col-xs-12 col-sm-12 col-lg-3 mt-2">
+
+        <div >
+          <h2 class="text-muted mt-4 fs-6">THE DEBATE</h2>
+          <div class="border-top border-5">
+            <div class="d-flex align-items-center  border-bottom pb-3">
+
+              <a href="#"><img class="p-3" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1673623/froma-harrop.png?w=63&h=63&f=2ba48b46c4bcf031c010ff486c04d55b" alt="froma-harrop"></a>
+              <div class="">
+                <a class="text-body text-decoration-none" href="#"><h6>Media Have Every Right to Cancel Trump</h6></a>
+                <span class="text-body">BY</span>
+                <a class="text-body text-decoration-none fw-bold" href="#">FROMA HARROP</a>
+              </div>
+
+            </div>
+          </div>
+        </div>
+
+        <div class="position-relative">
+          <div class="versus position-absolute rounded-circle bg-dark">
+            <p class="px-3 mt-2 text-muted fw-bolder">vs</p>
+          </div>
+          <div class="border-bottom border-5">
+            <div class="border-top mt-1 d-flex align-items-center justify-content-between pt-4">
+
+              <img class="p-3 order-2" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1673623/froma-harrop.png?w=63&h=63&f=2ba48b46c4bcf031c010ff486c04d55b" alt="froma-harrop">
```

Comment:
[required] As it was mentioned in the previous review, [all images should have a link] => so does this image: it should have a link

Reply:
We've now included the a tag for this image. @descholar-ceo